app/client/src/utils/utils.js

```diff
@@ -10,6 +10,21 @@ function chunkArray(array: any, chunkLength: number): Array<Array<any>> {
   return chunks;
 }
 
+function containsScriptTag(string: string) {
+  string = string
+    .toLowerCase()
+    .replace('%20', '')
+    .replace('%3c', '<')
+    .replace('%3e', '>')
+    .trim();
+  console.log('string: ', string);
```

Comment:
Just noticed a console.log, I think we can delete this line

Reply:
Good catch, I went a head and deleted that line.